### DISCUSSION TITLE
Configuring DNS for subdomain to point to external server

### DISCUSSION THREAD

**Discussion by anttituomola** [vercel/community#3141] [Help] [answered]
### Summary

Hi there,

I'm hosting my site on Vercel, but I've set up a Plausible Analytics server on a DigitalOcean droplet. I'm trying to use a subdomain (analytics.mydomain.com) to point to this server, so I added an A record in my DNS settings.

But, Vercel is giving me an error saying I have a conflicting DNS record and is asking me to use a CNAME record that points to Vercel. The problem is, I need the A record to point to my DigitalOcean server for the analytics subdomain.

Is there a way I can get analytics.mydomain.com to point to my Plausible server while keeping the rest of my DNS settings the same on Vercel? Any guidance would be appreciated!

Thanks!

### Steps to Reproduce

<img width="713" alt="2023-07-06 14_00_28-Project Settings – Dashboard – Vercel - Brave" src="https://github.com/vercel/community/assets/25209560/1a84d471-03c3-4e06-a72f-2ea1ff62f516">

**Comment by amyegan** [comment, marked as answer]
Hi @anttituomola. It looks like you're trying to add the `anaytics.mydomain.com` subdomain to your project, and that is likely the source of the error. Since you're pointing the subdomain to your DigitalOcean droplet, you should only need to set up the DNS record without adding the subdomain to the project.

**Reply by anttituomola** [reply]
You are correct; thank you so much for your help. Much appreciated!

**Reply by denizsult** [reply]
@anttituomola Hello, are you solved your problem. I have the same issue but cant handle it.

**Reply by anttituomola** [reply]
Yep, it was solved by using the actual domain, not the subdomain: mydomain.com instead of analytics.mydomain.com
